tests/golem/task/test_taskcomputeradapter.py

```diff
@@ -0,0 +1,361 @@
+from pathlib import Path
+from unittest import mock
+
+from golem_messages.message import ComputeTaskDef
+from twisted.internet import defer
+from twisted.trial.unittest import TestCase as TwistedTestCase
+
+from golem.task.envmanager import EnvironmentManager
+from golem.task.taskcomputer import TaskComputerAdapter
+from golem.task.taskserver import TaskServer
+from tests.factories.taskserver import ClientConfigDescriptor
+
+
+class TaskComputerAdapterTestBase(TwistedTestCase):
+
+    @mock.patch('golem.task.taskcomputer.IntStatsKeeper')
+    @mock.patch('golem.task.taskcomputer.TaskComputer')
```

Comment:
How about spec_set?
```suggestion
    @mock.patch('golem.task.taskcomputer.TaskComputer', mock.Mock(spec_set=TaskComputer))
```

same below